dwh/query_lib_test.py

```diff
@@ -16,7 +16,6 @@
 
 import query_lib
 
-
 class PatientQueryTest(unittest.TestCase):
```

Comment:
I am looking at the test failure to see if that is due to the permission issues we were expecting. But from the [test logs](https://pantheon.corp.google.com/cloud-build/builds/c2bda07a-e0a9-450a-8824-1953889cd414;step=14?project=cloud-build-fhir) it seems to me that is actually in these unit-tests before getting to running BigQuery queries. IIUC, you have changed the interface of `PatientQuery` and for example it does not have a `all_constraints_sql` method anymore. Can you please take a look @gdevanla?

Reply:
My bad. I did not update these tests. I have updated them. These tests should pass now.